menu/views.py

```diff
@@ -1,3 +1,42 @@
-from django.shortcuts import render
+from rest_framework import viewsets, filters, generics, status
 
-# Create your views here.
+from django_filters.rest_framework import DjangoFilterBackend
+from location.models import Location
+from menu.filters import CustomSearchFilter, CustomRangeFilter, perform_product_filtering
+from menu.serializers import LocationMenuSerializer, ProductMenuSerializer
+from product.models import Product
+from rest_framework.response import Response
+
+
+class ProductMenuView(viewsets.ReadOnlyModelViewSet):
```

Comment:
is this view needed?

Reply:
i want to left DRF version. If you whant I can torn off but not delete